### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by bklimey**
Add 'Support for multiple social handles' section

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -196,6 +192,15 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
 The features mentioned are:
+<!---
+- [Adding contacts with optional arguments](#add-contacts-with-optional-arguments)
+- [Finding contacts by keywords](#find-contacts-by-keywords)
+- [Deleting multiple contacts by keywords](#delete-by-keywords)
+- [Importing a JSON file](#import-json-file)
+- [Support for multiple social handles](#support-for-multiple-social-handles)
+- [Undoing / redoing command](#proposed-undoredo-feature)
+- [Input Suggestion](#input-suggestion)
+--->
```
Comment (kevinmingtarja) [inline docs/DeveloperGuide.md:203]: Can I ask what is this for?

**Review by kevinmingtarja — CHANGES_REQUESTED** [review]
Minor comment but otherwise LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bklimey) [reply docs/DeveloperGuide.md:203]: This is the commented out section of the previous version. I will remove it.

**Review by hsiaotingluv — APPROVED** [review]

**Review by kevinmingtarja — APPROVED** [review]
LGTM

**Pull request merged**